esmvaltool/reformat.py

```diff
@@ -0,0 +1,45 @@
+"""Port reformat.py from v1 to v2 (life hack)."""
+import fnmatch
+import os
+from subprocess import call
+
+
+def cmor_reformat(config, logvar):
+    """Run the oldskool v1 reformat scripts."""
+    logvar.info("Running the oldschool reformatting scripts.")
```

Comment:
`Running the CMORization scripts for the observations"
These will be actually v2 once ported.